streams/ua.m3u

```diff
@@ -1,19 +1,13 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="1Plus1Sport.ua" status="online",1+1 Спорт (720p) [Not 24/7]
 https://live-k2301-kbp.1plus1.video/sport/smil:sport.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="" status="online",4 канал (576p)
-http://95.67.106.10/hls/nta_ua_low/index.m3u8
 #EXTINF:-1 tvg-id="" status="online",4 канал (720p)
-http://95.67.106.10/hls/nta_ua_mid/index.m3u8
-#EXTINF:-1 tvg-id="" status="online",4 канал (1080p)
-http://95.67.106.10/hls/nta_ua_hi/index.m3u8
+http://95.67.106.10/hls/nta_ua.m3u8
```

Comment:
New link is not working:
<img width="645" alt="image" src="https://user-images.githubusercontent.com/7253922/155750564-96924a6f-5918-4f24-b150-dd5a5d360c7c.png">

Reply:
![Screenshot_20220225-174328.jpg](https://user-images.githubusercontent.com/30985701/155753600-ba8d0d40-bc33-4cb5-8d4e-5f020bf440c1.jpg)

Reply:
I don't know what exactly the problem is, but when the old links work without problems 
<img width="647" alt="image" src="https://user-images.githubusercontent.com/7253922/155833450-48d1797a-a6db-4f20-ac2c-1c02b601630d.png">
the new one still return an error
<img width="616" alt="image" src="https://user-images.githubusercontent.com/7253922/155833463-e9ff1eed-1fb3-48da-898f-07792baf804c.png">

Reply:
Let me see if this could be an user-agent issue.

Reply:
Just tested my Firefox user agent against the website with VLC, works.

Reply:
Unfortunately, I still have the same 404 in VLC.

Reply:
This is really weird. I guess i'll have to keep both the adaptive and the other links then.

Reply:
Added the original links back. Problem should be solved.